src/docs/product/accounts/quotas/index.mdx

```diff
@@ -140,7 +140,7 @@ After these checks are processed, the event counts toward your quota. It is acce
 
 ## Increasing Quotas
 
-Add to your quota at any time during your billing period, either by upgrading to a higher tier or by increasing your on-demand capacity. Once on-demand is activated, teammates with the "owner" organization permission level will receive a [notification](/product/alerts/notifications/#quota) email. Our available plans suit most individual and business needs, and Sentry is designed to handle large throughput. If your team needs more, we’re happy to help. Reach out to our sales team at [sales@sentry.io](mailto:sales@sentry.io) to increase capacity.
+Add to your quota at any time during your billing period, either by upgrading to a higher tier or by increasing your on-demand capacity. Once on-demand is activated, teammates with the "owner" organization permission level will receive a [notification](/product/alerts/notifications/#quota-notifications) email. Our available plans suit most individual and business needs, and Sentry is designed to handle large throughput. If your team needs more, we’re happy to help. Reach out to our sales team at [sales@sentry.io](mailto:sales@sentry.io) to increase capacity.
```

Comment:
I don't think that's always true, as on-demand emails only send for older plans and at specific thresholds ($50, $100, etc.). We're planning to work on notifications more next quarter so we could update the docs then.

```suggestion
Add to your quota at any time during your billing period, either by upgrading to a higher tier or by increasing your on-demand capacity. Our available plans suit most individual and business needs, and Sentry is designed to handle large throughput. If your team needs more, we’re happy to help. Reach out to our sales team at [sales@sentry.io](mailto:sales@sentry.io) to increase capacity.
```